Description: use dh_installexamples to install examples, not the Makefiles.
Author: Ryan Kavanagh <ryanakca@kubuntu.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2016-11-27
--- a/BASIC256.pro
+++ b/BASIC256.pro
@@ -111,7 +111,7 @@
 	########
 	exampleFiles.files		=	./Examples
 	exampleFiles.path		=	/usr/share/basic256
-	INSTALLS				+=	exampleFiles
+	# INSTALLS				+=	exampleFiles
 
 	helpHTMLFiles.files		=	./wikihelp/help
 	helpHTMLFiles.path		=	/usr/share/basic256
